### PR TITLE
monitoring: config for 0.1.0 release

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: manifests/plugin/monitoring.pp [hunk 1]
```diff
@@ -21,8 +23,9 @@
   Boolean $enabled = $::foreman_proxy::plugin::monitoring::params::enabled,
   Optional[String] $group = $::foreman_proxy::plugin::monitoring::params::group,
   Foreman_proxy::ListenOn $listen_on = $::foreman_proxy::plugin::monitoring::params::listen_on,
-  String $provider = $::foreman_proxy::plugin::monitoring::params::provider,
+  Array $providers = $::foreman_proxy::plugin::monitoring::params::providers,
```
Comment (ekohl) [inline manifests/plugin/monitoring.pp]: This can be ```Array[String]```.

File: manifests/plugin/monitoring/icinga2.pp [hunk 2]
```diff
@@ -27,6 +29,7 @@
 class foreman_proxy::plugin::monitoring::icinga2 (
   Boolean $enabled = $::foreman_proxy::plugin::monitoring::icinga2::params::enabled,
   String $server = $::foreman_proxy::plugin::monitoring::icinga2::params::server,
+  Integer $api_port = $::foreman_proxy::plugin::monitoring::icinga2::params::api_port,
```
Comment (ekohl) [inline manifests/plugin/monitoring/icinga2.pp]: You can further restrict this to ```Integer[0, 65535]```.

**Comment by timogoebel** [comment]
@ekohl : Thanks for your comments. Fixed them.

**Comment by domcleal** [comment]
Merged, thanks @timogoebel. Sorry for the delay.

**Pull request merged**